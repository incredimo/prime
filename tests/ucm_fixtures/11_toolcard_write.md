```ucm set#sT1 path=".tools/loggrep/toolcard.json" mode="write"
{"name":"loggrep","version":"1.0.0","summary":"parse logs",
 "inputs":{"$schema":"..."},
 "outputs":{"$schema":"..."},
 "runner":{"type":"sh","entry":"impl.sh","timeout_s":5,"stdin":false},
 "resources":{"network":false,"filesystem":"read-only","cpu_limit":1.0,"mem_mb":128},
 "relevance":{"triggers":[{"if_task_contains":["log"]}],"search_keywords":["logs","log parsing","grep"]},
 "tags":["log"],"owner":"prime","created":"2025-08-30","changelog":[{"version":"1.0.0","notes":"initial"}]}
```