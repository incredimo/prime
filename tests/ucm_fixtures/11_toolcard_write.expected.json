{
  "id": "sT1",
  "verb": "set",
  "attrs": { "path": ".tools/loggrep/toolcard.json", "mode": "write" }
}